test/rootfs/setup.sh

```diff
@@ -227,6 +227,13 @@ git clone https://github.com/sstephenson/bats.git "${tmpdir}/bats"
 curl -fsLo "/usr/local/bin/jq" "http://stedolan.github.io/jq/download/linux64/jq"
 chmod +x "/usr/local/bin/jq"
 
+# install protobuf compiler
+curl -sL https://github.com/google/protobuf/releases/download/v3.0.0/protoc-3.0.0-linux-x86_64.zip > ~/protoc.zip
+trap "rm ~/protoc.zip" EXIT
```

Comment:
Yeh so ideally we would run this in a job and allow flynn-host to do the cleanup :smile:.

I actually have a branch where I did that (I think that is when I realised the trap issue), so that can be the proper fix for this.